### ISSUE TITLE
How to extract feature in VGG16 for a test image? python - keras

### ISSUE THREAD

**Issue by mrazimi** [keras-team/keras#10720]
Sorry I am beginner, and I have a question:
Keras has a way to extract the features of a pretrained model:
vgg_model = VGGFace() # pooling: None, avg or max
out = vgg_model.get_layer('fc7').output
vgg_model_fc7 = Model(vgg_model.input, out)
img = image.load_img(imagepath, target_size=(224, 224))
x = image.img_to_array(img)
x = np.expand_dims(x, axis=0)
x = x[:, :, :, ::-1]
vgg_model_fc7_preds = vgg_model_fc7.predict(x)
print(vgg_model_fc7_preds[0])
Is predict (before last line) a tool for feature extraction (for getting output of layer) or for prediction of user? 
If it is for feature extraction, why name it "predict" ?

**Comment by Dref360** [comment]
Hi!
This issue isn't related to a bug/enhancement/feature request or other accepted types of issue.

To ask questions, please see the following resources : 
* https://keras.io/
* https://keras.io/#support
* https://gitter.im/Keras-io/Lobby

Thanks!

If you think I made a mistake, please re-open this issue.
